libs/langchain/langchain/vectorstores/weaviate.py

```diff
@@ -447,9 +489,6 @@ def from_texts(
 
             batch.flush()
 
-        relevance_score_fn = kwargs.get("relevance_score_fn")
-        by_text: bool = kwargs.get("by_text", False)
```

Comment:
@hsm207 this default to True in init and False in from_texts — is that on purpose?

Reply:
yes. it is on purpose. 

This issue has the context: https://github.com/langchain-ai/langchain/issues/4742

Reply:
are you referring to this comment?

> can we have a way to toggle between the two? eg keep both functionality and let the user decide?

i get why the parameter exists, but why does it have different defaults depending on how you instantiate?

Reply:
I can't remember what the exact reason was anymore but not doing that will cause some examples in the demo notebook to break. 

Context: https://github.com/langchain-ai/langchain/pull/5201

Reply:
we can always update notebook if needed — is there one that you'd strongly prefer to be default?

Reply:
I don't have a strong preference for a default value for `by_text`.